tests/app/goss.yaml

```diff
@@ -18,11 +18,12 @@ http:
       - '/^pool: /'
       - '/^process manager: /'
 {{ end }}
+{{ if (not (eq (or .Vars.app "") "nuxt"))}}
   # App
   http://localhost:
     status: 200
-{{ if .Vars.app }}
+  {{ if .Vars.app }}
```

Comment:
Are you sure there are no indentation issues doing this ?
Maybe something like `{{-‘ should handle them, if any...